src/features/surveys/components/Questionnaire.tsx

```diff
@@ -1,12 +1,12 @@
-import React, { memo } from "react";
+import React, { memo, ReactElement } from "react";
 import ReactDOMServer from "react-dom/server";
-import * as Survey from "survey-react";
 import { compact, groupBy, sortBy, toString } from "lodash";
+import * as Survey from "survey-react";
 import * as widgets from "surveyjs-widgets";
-import CheckCircleIcon from "@mui/icons-material/CheckCircle";
-import { Typography } from "@mui/material";
+import { Survey as SurveyInterface } from "../models/Survey.types";
+import { LocalStorageNames } from "../constants/localStorage";
 
-import api from "../capableApi/index";
+import api from "../../../capableApi/index";
```

Comment:
I think you can import it directly
```suggestion
import api from "capableApi/index";
```